bundle/DependencyInjection/Configuration.php

```diff
@@ -50,5 +50,10 @@ protected function addConfiguration(ArrayNodeDefinition $rootNode): void
             ->booleanNode('render_missing_field_info')
                 ->info('Whether to render useful debug information in place of a missing field')
             ->end();
+        $systemNode
+            ->booleanNode('use_param_converters')
```

Comment:
Why do we need this? Isn't the purpose of `supports` method to declare what typehints param converter supports?

Moreover, I don't think this "overrides" eZ param converters. They can live side by side without issues.